gcloud/_apitools/http_wrapper.py

```diff
@@ -372,6 +321,53 @@ def _MakeRequestNoRetry(http, http_request, redirections=5,
     return response
 
 
+def MakeRequest(http, http_request, retries=7, max_retry_wait=60,
```

Comment:
So that I can pass in the new `wo_retry_func` argument when testing.  That argument defaults to `_MakeRequestNoRetry`, which must therefore be defined before `MakeRequest`.